.travis.yml

```diff
@@ -12,12 +19,6 @@ env:
         - DISTRIB="conda" PYTHON_VERSION="3.4"
 
 before_install:
-    - if [ "$DISTRIB" == "ubuntu" ]; then
-         sudo apt-get update -qq;
-         sudo apt-get install -qq python-numpy python-matplotlib
-         python-setuptools;
-      fi
-
```

Comment:
sudo:false is to use the new travis Container-Based Infrastructure which allows for faster builds. And all this apt-get packages are cached, so very fast install. Speed gain is really noticeable.

Yes these get installed in the conda builds as well but don't mix at all since conda has a virtual environment.